packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -0,0 +1,62 @@
+import DotGovIcon from './DotGovIcon';
+import HttpsIcon from './HttpsIcon';
+import LockIcon from './LockIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import UsaFlagIcon from './UsaFlagIcon';
+
+const UsaBanner = function (props) {
+  const bannerHeader = props.isBannerOpen
+    ? 'ds-c-usa-banner__header ds-c-usa-banner__header--expanded'
+    : 'ds-c-usa-banner__header';
+
+  return (
+    <div className="ds-c-usa-banner">
+      <header className={bannerHeader}>
+        <UsaFlagIcon />
```

Comment:
This icon is great! how did you end up converting to SVG?

Reply:
Oh i see, we already had this SVG in the images

Reply:
Instead of redefining our .svg files as react components, I think we should consider using this: 
https://github.com/airbnb/babel-plugin-inline-react-svg

Once we install that dependency to our scripts package, and we update our babel.config.js, we should be able to directly import our svg icon files and use them as React components

Reply:
This could also be a very promising solution for moving away from using icons as background images

Reply:
It would also be really nice to update our svg files to be cleaner and formatted well, so they are easy to read and use.